www/about/faq.spt

```diff
@@ -75,7 +75,7 @@ subhead = _("Frequently Asked Questions")
         <ol>
 
             <li><a href="https://gratipay.com/bank-account.html">Connect a
-                bank account</a>. Funds will be deposited every Friday (no fee;
+                bank account</a>. Funds will be deposited every Friday ($10 minimum; no fee;
```

Comment:
MINIMUM_CREDIT is a good stuff to have here too, so that we won't have to keep track on that. Although I'd change the name of this constant to MINIMAL_PAYOUT.